lib/time.ml

```diff
@@ -2,9 +2,9 @@
  * http://www.ocsigen.org/lwt
  * Module Lwt_mirage, based on Lwt_unix
  * Copyright (C) 2010 Anil Madhavapeddy
- * Copyright (C) 2005-2008 J�r�me Vouillon
```

Comment:
This hunk looks unnecessary

Reply:
Right, I'm not sure how that happened.

Reply:
My editor most likely changed some encoding automatically. That's fixed.